src/core/operators/file_processing_op.py

```
@@ -0,0 +1,344 @@
+import asyncio
+from typing import Dict, Any, Optional, List, TypedDict, Annotated
+import yaml
+import base64
+import io
+from PIL import Image
+
+import numpy as np
+from langchain_core.messages import BaseMessage, HumanMessage, AIMessage, SystemMessage
+from langgraph.graph import StateGraph, START, END
+from langgraph.graph.message import add_messages
+from langchain_core.messages import BaseMessage, HumanMessage, AIMessage, SystemMessage
```

Ruff (0.12.2)

Redefinition of unused `BaseMessage` from line 19

Remove definition: `BaseMessage`

(F811)